Extending/Section-Trees/tree-actions-v8.md

```diff
@@ -52,42 +52,34 @@ When you pull back the AssignedPermissions for a user on a particular item, it i
-| ®      | create               | False                      | add                 | javascript:actionNewRelationType();                                                                                                     |
-| V      | ActionRestore        | False                      |                     | This action is invoked when the content item is to be restored from the recycle bin                                                     |
-| -      | Action Null          | False                      |                     | This is used internally to assign no permissions to a node for a user and shouldn't be used in code                                     |
-| ,      | ContextMenuSeparator | False                      |                     | Used to define context menu separator items. This should not be used directly in any code except for creating menus              |
+Here is a list of the tree actions and associated user permission codes shipped by Umbraco CMS and add-on projects (such as Umbraco Deploy), as well as those used by some community packages.
+
+If building a package or adding custom tree actions to your solution, it's important to pick a permission letter that doesn't clash with one of these.
+
+If you have created a package using a custom tree action, please consider providing an update to this documentation page via a PR to the [documentation repository](https://github.com/umbraco/UmbracoDocs), such that other developers can discover and avoid using the same permission letter.
+
+|Type|Alias|Letter|Can Be Permission Assigned|
+|-|-|-|-|
+|Umbraco.Cms.Core.Actions.ActionAssignDomain|assignDomain|I|True|
+|Umbraco.Cms.Core.Actions.ActionBrowse|browse|F|True|
+|Umbraco.Cms.Core.Actions.ActionCopy|copy|O|True|
+|Umbraco.Cms.Core.Actions.ActionCreateBlueprintFromContent|createblueprint|‹|True|
+|Umbraco.Cms.Core.Actions.ActionDelete|delete|D|True|
+|Umbraco.Cms.Core.Actions.ActionMove|move|M|True|
+|Umbraco.Cms.Core.Actions.ActionNew|create|C|True|
+|Umbraco.Cms.Core.Actions.ActionNotify|notify|N|True|
+|Umbraco.Cms.Core.Actions.ActionProtect|protect|P|True|
+|Umbraco.Cms.Core.Actions.ActionPublish|publish|U|True|
+|Umbraco.Cms.Core.Actions.ActionRestore|restore|V|False|
+|Umbraco.Cms.Core.Actions.ActionRights|rights|R|True|
+|Umbraco.Cms.Core.Actions.ActionRollback|rollback|K|True|
+|Umbraco.Cms.Core.Actions.ActionSort|sort|S|True|
+|Umbraco.Cms.Core.Actions.ActionToPublish|sendtopublish|H|True|
+|Umbraco.Cms.Core.Actions.ActionUnpublish|unpublish|Z|True|
+|Umbraco.Cms.Core.Actions.ActionUpdate|update|A|True|
+|Umbraco.Deploy.UI.Actions.ActionDeployRestore|deployRestore|Q|True|
+|Umbraco.Deploy.UI.Actions.ActionDeployTreeRestore|deployTreeRestore|Ψ|True|
+|Umbraco.Deploy.UI.Actions.ActionPartialRestore|deployPartialRestore|Ø|True|
+|Umbraco.Deploy.UI.Actions.ActionQueueForTransfer|deployQueueForTransfer|TBC|True|
```

Comment:
The namespaces in Umbraco 8 are a little bit different:
```suggestion
|Umbraco.Web.Actions.ActionAssignDomain|assignDomain|I|True|
|Umbraco.Web.Actions.ActionBrowse|browse|F|True|
|Umbraco.Web.Actions.ActionCopy|copy|O|True|
|Umbraco.Web.Actions.ActionCreateBlueprintFromContent|createblueprint|ï|True|
|Umbraco.Web.Actions.ActionDelete|delete|D|True|
|Umbraco.Web.Actions.ActionMove|move|M|True|
|Umbraco.Web.Actions.ActionNew|create|C|True|
|Umbraco.Web.Actions.ActionProtect|protect|P|True|
|Umbraco.Web.Actions.ActionPublish|publish|U|True|
|Umbraco.Web.Actions.ActionRestore|restore|V|False|
|Umbraco.Web.Actions.ActionRights|rights|R|True|
|Umbraco.Web.Actions.ActionRollback|rollback|K|True|
|Umbraco.Web.Actions.ActionSort|sort|S|True|
|Umbraco.Web.Actions.ActionToPublish|sendtopublish|H|True|
|Umbraco.Web.Actions.ActionUnpublish|unpublish|Z|True|
|Umbraco.Web.Actions.ActionUpdate|update|A|True|
|Umbraco.Deploy.UI.Actions.ActionDeployTreeRestore|deployTreeRestore|Ψ|True|
|Umbraco.Deploy.UI.Actions.ActionDeployRestore|deployRestore|Q|True|
|Umbraco.Deploy.UI.Actions.ActionPartialRestore|deployPartialRestore|Ø|True|
|Umbraco.Deploy.UI.Actions.ActionQueueForTransfer|deployQueueForTransfer|N|True|
```

I also noticed that the Deploy actions were added before the CMS ones, so that will probably result in 8.18 using the opposite permission for the letter `N` (and thus `Queue for transfer` will still work, but `Notification` won't).